pykafka/simpleconsumer.py

```diff
@@ -295,6 +308,9 @@ def consume(self, block=True):
         :param block: Whether to block while waiting for a message
         :type block: bool
         """
+        if not self._running:
+            return None
```

Comment:
This check is already handled below in L331, where it raises an exception instead, so these two lines should probably go?